openedx/core/djangoapps/catalog/utils.py

```diff
@@ -394,13 +394,14 @@ def get_fulfillable_course_runs_for_entitlement(entitlement, course_runs):
             course_id=course_id
```

Comment:
Can you also update the comment on this method.  It is poorly worded, it went through a lot of iterations.

Reply:
Any suggestion?

Reply:
Something like the following maybe?

``` 
Looks through the list of course runs and returns the course runs that be applied to the entitlement.

Arguments:
    entitlement (CourseEntitlement): The CourseEntitlement that we would like to apply a course run to.
    course_runs (list): List of course run keys that we would like to apply to the entitlement.

Returns
    list: A list of sessions that a user can apply to the provided entitlement.
```